src/engine/routing_algorithms/map_matching.cpp

```diff
@@ -155,13 +145,25 @@ SubMatchingList mapMatching(SearchEngineData<Algorithm> &engine_working_data,
     for (auto t = initial_timestamp + 1; t < candidates_list.size(); ++t)
     {
 
+        const auto step_time =
```

Comment:
This will cause a SIGSEV if `trace_timestamps` is empty (`use_timestamps == false`)